ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -270,6 +270,7 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue",
+    "Publish": "Y",
```

Comment:
I think based on [this](https://tools.hmcts.net/confluence/display/RCCD/CCD+Definition+Glossary+for+Setting+up+a+Service+in+CCD#CCDDefinitionGlossaryforSettingupaServiceinCCD-ccdDefTabCaseEvent) it may need to be 'Yes' rather than 'Y' (unless the excel generator is converting them for us)?

Reply:
Think it can be 'Y','Yes','T' or 'True'! https://tools.hmcts.net/confluence/display/RCCD/Work+Allocation+Phase+2+Scope+of+Delivery